.github/workflows/build.yaml

```diff
@@ -145,7 +145,7 @@ jobs:
                     dependency-versions: "${{ matrix.dependencies }}"
 
             -   name: "Run phpunit"
-                run: "composer phpunit"
+                run: "composer run-script phpunit"
```

Comment:
No need to add the `run-script`. It works the same?

Reply:
:) no need for line break in PHP source code

Explicit programming helps others read your code and help you to think.

Reply:
So we agree on that statement, however, it seems we also agree it's good to simplify some things. You have removed `vendor/bin` from the composer scripts, which contradicts this change :)

Reply:
- `vendor` can be customized
- Composer pushes bin-dir in front of `PATH`
- it is irritating to see `vendor/bin/` in every line as these line as these are already Composer scripts

Reply:
`composer phpunit` could be

1. a built-in Composer command
1. a Composer command from a plugin
1. a Composer script
1. may be something else ... we do not know as it is not written here